fs-storage/examples/cli.rs

```diff
@@ -107,5 +125,87 @@ fn write_command(args: &[String], path: &str) -> Result<()> {
             }
         }
     }
+    fs.write_fs().expect("Failed to write to file");
+    Ok(())
+}
+
+fn folder_read_command(args: &[String], path: &str) -> Result<()> {
+    let keys = if args.len() > 3 {
```

Comment:
I think this should get it working.
```suggestion
    let keys = if args.len() > 4 {
```

Reply:
Doesn't `clap` handle such details automatically?

Reply:
Currently, we are not using `clap` in the example